### ISSUE TITLE
`.env` variables taking precedent over already establish env variables

### ISSUE THREAD

**Issue by PatrickAlphaC** [eth-brownie/brownie#1041]
### Environment information

* `brownie` Version: 1.14.4
* Python Version: 3.8.0
* OS: osx

### What was wrong?

The variables in the `.env` file are overriding the variables in the shell.

Typically, [`dotenv`](https://pypi.org/project/python-dotenv/) doesn't override variables that have already been established unless explicitly stated. 

### How can it be fixed?

I think this would go well with the discussion in https://github.com/eth-brownie/brownie/issues/1031

Basically, using the `load_dotenv()` instead of manually assigning with `dotenv_values()`

Open to discussion of course!

**Comment by omarish** [comment]
Yup, that was intentional and we could change it to work either way. We could change the priority so that `.env` variables are set iff they are not already set in `os.environ`. Thoughts?

**Comment by PatrickAlphaC** [comment]
I like that :) 

@iamdefinitelyahuman?

**Comment by spinoch** [comment]
> We could change the priority so that .env variables are set iff they are not already set in os.environ

I think this makes sense as it allows for overriding those .env variables when desired. Docker-compose also does this in this way
